### DISCUSSION TITLE
如何引用时作者名之间用“和”而不是“and”连接？

### DISCUSSION THREAD

**Discussion by DBY2-W00** [stone-zeng/fduthesis#255] [Q&A] [answered]
设置如下
bib-backend = bibtex,
bib-style=author-year,
cite-style=numbers,

引用参考文献时\citet*{XXX}如果是两个作者那么就显示 A and B[序号]
如何改成A和B[序号]？

**Comment by sikouhjw** [comment, marked as answer]
1. `bibtex` 下 `cite-style` 是无效选项
2. `texdoc biblatex-gb7714-2015` 中没有你这种写法
   ![图片](https://user-images.githubusercontent.com/49770925/166898511-c6157d30-4cc5-4deb-af12-085c42626fea.png)
3. `biblatex` 中可以用 `\setlocalbibstring{andincite}{和}` 来临时改变，但无法输出多个作者，因此在 `biblatex` 下这种需求不能直接满足
4. 我对 `bibtex` 的 bst 不熟，不知道该如何修改

**Comment by iamforerunner** [comment]
\DefineBibliographyStrings{english}{   % 针对英文两个作者，弄出 Zhang and Li, 2020，而非 Zhang, Li, 2020
        andincite         = { and },
        andincitecn       = {和},
        andothersincite = {et al.\isdot},
        % andothersincitecn   = {等{\adddot}} %adddot才能避开标点追踪
        andothersincitecn   = {等}
}

试试这段code
